cluster-autoscaler/core/scaleup/orchestrator/orchestrator.go

```diff
@@ -119,32 +119,53 @@ func (o *ScaleUpOrchestrator) ScaleUp(
 		}
 	}
 
+	var nodeGroupPoolList [][]cloudprovider.NodeGroup
```

Comment:
Could you add some unit-tests to cover the usage of this processor?

Reply:
Done

Reply:
Just added a UT similar to the one including `NodeGroupListProcessor`. This UT might get changed in future.